install_pim/docker/installation_docker.rst

```diff
@@ -71,8 +74,8 @@ You need to get a PIM Enterprise Standard archive from the Partners Portal. See
     $ cd pim-enterprise-standard
     $ docker run -ti -u www-data --rm \
         -e COMPOSER_MEMORY_LIMIT=4G \
-        -v $(pwd):/srv/pim -v ~/.composer:/var/www/.composer -v ~/.ssh:/var/www/.ssh -w /srv/pim \
```

Comment:
I remove the default mount and it will be up to the devs to mount a local cache or not, with a path depending on their machine

Reply:
Why do change, and not setup composer cache because composer is just run once in a year?
Should be documented somehow?

Reply:
Because the cache on the host (the dev machine) can be in several locations:
- ~/.cache/composer : the standard installation for a fresh composer v2
- ~/.composer/cache : for users coming from old composer v1
- Anywhere else if they configured their env

Lastly, it can also be nowhere, like for my machine relying only on docker and having no php nor composer installed locally.

Reply:
The last argument is indeed very good. I saw it happening a lot of time, and this lead to Docker creating the directory on the host, accessible only for root, ending in failure as Composer cannot write in it...